### DISCUSSION TITLE
error - SyntaxError: Unexpected token u in JSON at position 0     at JSON.parse (<anonymous>) after updating to nextjs 13.3

### DISCUSSION THREAD

**Discussion by AmanVarshney01** [vercel/next.js#48089] [Help]
### Summary

it worked absolutely fine in 13.2.4 but after updating to 13.3.0 showing
error - SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)

**Comment by EnricoBaivo** [comment]
Did you find the reason for the error, i also encounter the same problem.

**Reply by ethanlamm** [reply]
when I used "use client" to mark the component to be the client component, the same error occurred. Just don't use the 13.3.0v, go back to the 13.2.4, problem solved😁

**Comment by luuap** [comment]
Upgrading to 13.3.1-canary.4 fixed it for me. If it still doesn't work for you maybe check out #48070.
